benchmarks/benchmarks/stats.py

```diff
@@ -3,15 +3,17 @@
 import warnings
 
 import numpy as np
+from numpy.random import rand
 
 try:
     from scipy.stats import anderson_ksamp
+    from scipy.stats import fisher_exact
+    import scipy.stats as stats
```

Comment:
Will do the change. Actually there was a merge conflict as some other commit also added ``stats.py`` to the benchmarks.